README.md

```diff
@@ -101,20 +100,20 @@ A significant number of experts, editors, and authors have worked to assemble an
 it.
```

Comment:
@jlevy for "Why this Guide?" consider:
"The authors of this Guide want to help everyone get a fair shake.
We hope to save you time and effort when navigating this complex topic, help you to
understand what can and can’t be controlled when it comes to equity compensation, and how
best to think about some very important decisions."